README.md

```diff
@@ -1,2 +1,201 @@
-# bookstore
-This is a website that displays the list of books that we can update by adding a new book or remove a selected book. Built with React and Redux
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Bookstore\] ](#-bookstore-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Bookstore] <a name="about-project"></a>
+
+**[Bookstore]** This is a website that displays the list of books that we can update by adding a new book or remove a selected book. Built with React and Redux
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Styling</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[list_of_books]**
+- **[add-book]**
+- **[delete-book]**
+- **[edit-book-details]**
+- **[update-progress]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming Soon](Coming-Soon)
```

Comment:
- [ ] **[OPTIONAL]** You did a great job working on the documentation of your project! 🚀 I'm marking this as optional because it is the only requirement that's missing to approve, please consider removing the link syntax of your live demo section and write "Coming Soon" as text, currently there's a broken link that can mislead people browsing your project! 🤓 🔧 

```suggestion
## 🚀 Live Demo <a name="live-demo"></a>

- Coming Soon
```